osu.Game.Tournament/Screens/Gameplay/Components/MatchRoundDisplay.cs

```diff
@@ -4,21 +4,25 @@
 using osu.Framework.Allocation;
 using osu.Framework.Bindables;
 using osu.Game.Graphics;
+using osu.Game.Graphics.UserInterfaceFumo;
 using osu.Game.Tournament.Components;
+using osu.Game.Tournament.Localisation;
 using osu.Game.Tournament.Models;
 using osuTK.Graphics;
 
 namespace osu.Game.Tournament.Screens.Gameplay.Components
 {
-    public partial class MatchRoundDisplay : TournamentSpriteTextWithBackground
+    public partial class MatchRoundDisplay : FumoSpriteText
     {
         private readonly Bindable<TournamentMatch?> currentMatch = new Bindable<TournamentMatch?>();
 
         public MatchRoundDisplay()
         {
-            Background.Alpha = 0;
-            Text.Colour = Color4.White;
-            Text.Font = OsuFont.Torus.With(weight: FontWeight.Bold, size: 72);
+            BorderColour = Color4.White;
+            BackgroundColour = FumoColours.SeaBlue.Regular;
+            TextColour = Color4.White;
+            BorderThickness = 5;
+            Text.Font = OsuFont.Torus.With(weight: FontWeight.SemiBold, size: 72);
```

Comment:
```suggestion
            BorderThickness = 3;
            Text.Font = OsuFont.Torus.With(weight: FontWeight.SemiBold, size: 55);
```

Consider reducing the size of this component slightly. The original was too large and shouldn't be that visually dominant.